source/utils.js

```diff
@@ -69,6 +69,13 @@ function getStaticModule(file, testPath, errorMessage) {
     mod = mod.replace(/\\/g, '/');
   }
 
+  /*
```

Comment:
While console.log is still a powerful debug tool :-). I would like to avoid having comments in the code.

Reply:
I will remove this.

Reply:
This is what I used to debug the path that led to use `upath`